jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -659,10 +661,15 @@ public void afterTaskUpdatedEvent(TaskEvent event) {
                 return;
             }
 
-            if((ti.getDescription() != null && !ti.getDescription().equals(auditTaskImpl.getDescription()))
-                    || (ti.getDescription() == null && auditTaskImpl.getDescription() != null)){
+            if ((ti.getDescription() != null && !ti.getDescription().equals(auditTaskImpl.getDescription()))
+                    || (ti.getDescription() == null && auditTaskImpl.getDescription() != null)) {
                 String message = getUpdateFieldLog("Description", auditTaskImpl.getDescription(), ti.getDescription());
 
+                if (message != null && message.length() > TASK_DESCRIPTION_LENGTH) {
```

Comment:
this should not be required. it should be trimmed already in the task.

Reply:
Hi @elguardian,  TaskEvent message is getting persisted in the TaskEvent table and here task event message build as  "Updated Description from: 'old value' to:new value' ". If new value of task description is 255 chars, overall task event message length is more than 255 chars  so we have to trim the message as well.

Reply:
that is the reason it should be done in some other place. you will have a description that won't match database.

Reply:
I agree task description in TaskEvent wont match with task description in other tables, but message field from TaskEvent is not used anywhere else and actual value of TaskDescription is captured and in sync in Task and AuditTaskImpl table. If we dont update the TaskEvent.message then we have to trim the taskDescription more than defined length.  I can modify warn message and capture the task event message before truncating.

Reply:
@abhijithumbe task event is used in emitters... anycase there is much time for the cut off. I will let this slip